cmd/test-utils_test.go

```diff
@@ -1774,6 +1771,31 @@ func initAPIHandlerTest(obj ObjectLayer, endpoints []string) (string, http.Handl
 	return bucketName, f, nil
 }
 
+// prepare test backend.
+// create FS/XL bankend.
+// return object layer, backend disks.
+func prepareTestBackend(instanceType string) (ObjectLayer, []string, error) {
+	switch instanceType {
+	// Total number of disks for FS backend is set to 1.
+	case FSTestStr:
+		obj, disk, err := prepareFS()
+		if err != nil {
+			return nil, nil, err
+		}
+		return obj, []string{disk}, nil
+	// Total number of disks for XL backend is set to 16.
+	case XLTestStr:
+		return prepareXL()
+	default:
```

Comment:
This can be achieved simply by,
```go
switch instanceType {
case XLTestStr:
     return prepareXL()
default:
     obj, disk, err := prepareFS()
     if err != nil {
	return nil, nil, err
     }
     return obj, []string{disk}, nil
}
```